src/smk/feature-list/panel-feature-list.html

```diff
@@ -22,7 +22,7 @@
     >
         <!-- v-on:mouseenter="$$emit( 'hover', { features: layer.features } )" -->
         <!-- v-on:mouseleave="$$emit( 'hover', { features: null } )" -->
-        <h2>{{ layer.title }}<span class="smk-count" v-if="layer.features.length > 1">({{ layer.features.length }})</span></h2>
+        <!-- <h2>{{ layer.title }}<span class="smk-count" v-if="layer.features.length > 1">({{ layer.features.length }})</span></h2> -->
```

Comment:
this line should not be commented out. the task in jira was to remove the layer title on the feature detail panel, which is already done. The layer title in feature list is OK as it is

Reply:
the requirement also includes (as discussed with Denise) the popup that comes up with multiple features are selected at the same time, which currently has the "Feature #1, Feature#2..." list .

Reply:
removing this as per clarification from Denise